packages/nix/polymc/default.nix

```diff
@@ -69,11 +96,13 @@ mkDerivation rec {
   };
 
   postInstall = ''
```

Comment:
We need to add a `rm $out/share/applications/*` first here, due to the fact that the Cmake install puts a broken desktop file at `$out/share/applications/org.polymc.PolyMC.desktop` this is why I had a substitute in installPhase for the desktop file to patch the Exec= line

Reply:
Thanks but check line 100. Already overwriting that.

Reply:
oops didn't see that in the PR, my bad